apps/chat-e2e/src/ui/webElements/entityTree/sidebar/approveRequiredPrompts.ts

```diff
@@ -11,4 +12,14 @@ export class ApproveRequiredPrompts extends Folders {
       EntitySelectors.prompt,
     );
   }
+
+  public async expandApproveRequiredFolder(
```

Comment:
current methods is repeated in `ApproveRequiredConversationsTree` class.

Reply:
done